mne/cov.py

```diff
@@ -1165,7 +1170,7 @@ def prepare_noise_cov(noise_cov, info, ch_names, rank=None,
         if rank_meg is None:
             if len(C_meg_idx) < len(pick_meg):
                 this_info = pick_info(info, C_meg_idx)
-            rank_meg = _estimate_rank_meeg_cov(C_meg, this_info, scalings_)
+            rank_meg = _estimate_rank_meeg_cov(C_meg, this_info, scalings)
```

Comment:
what was wrong with this? This way you actually pass the original defaults, the other version produeced a copy, by updating from defaults into the passed dict, which I would prefer.